### PR TITLE
feat: light header theme was added to every page having the www-page …

### PR DESCRIPTION
**Pull request by roger-in-kiva**
Light header theme was added to every page having the www-page component

### CODE REVIEW

File: src/pages/15Years/15Years.vue [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default {
 	},
 	data() {
 		return {
-			headerTheme: fifteenYearHeaderTheme,
+			headerTheme: lightHeader,
```
Comment (mcstover) [inline src/pages/15Years/15Years.vue:56]: Please revert this page.

File: src/components/WwwFrame/TheHeader.vue [hunk 2]
```diff
@@ -60,7 +60,7 @@
 							<img
 								:src="profilePic"
 								alt="My portfolio"
-								class="fs-mask"
+								class="fs-mask tw-inline-block"
```
Comment (mcstover) [inline src/components/WwwFrame/TheHeader.vue:63]: Do these classes actually work in our current main branch? The `kv-tailwind` class would have to be on a parent in the current state...they will work fine in the next version...

File: src/pages/Homepage/iwd/IWDHomepage.vue [hunk 3]
```diff
@@ -352,7 +352,7 @@ export default {
 	},
 	data() {
 		return {
-			headerTheme: iwdHeaderTheme,
+			headerTheme: lightHeader,
```
Comment (mcstover) [inline src/pages/Homepage/iwd/IWDHomepage.vue:355]: Please revert this page.

File: src/pages/Homepage/wrd/WRDHomepage.vue [hunk 4]
```diff
@@ -231,7 +231,7 @@ export default {
 	},
 	data() {
 		return {
-			headerTheme: wrdHeaderTheme,
+			headerTheme: lightHeader,
```
Comment (mcstover) [inline src/pages/Homepage/wrd/WRDHomepage.vue:234]: Please revert this page.

File: src/pages/LendingStats/LendingStatsPage.vue [hunk 5]
```diff
@@ -82,6 +85,7 @@ export default {
 	},
 	data() {
 		return {
+			headerTheme: lightHeader,
```
Comment (mcstover) [inline src/pages/LendingStats/LendingStatsPage.vue:88]: Please revert this page. This is legacy stack and all of it's siblings in Portfolio have a "Green" header.

**Comment by mcstover** [comment]
@roger-in-kiva @emuvente @ryan-ludwig I've flagged a request to revert pages that had their own themes for the time being.

What do you all think about converting the Borrower Application and Settings pages to use a white header. Seems like maybe these should remain green for now due to their siblings being green...

File: src/components/WwwFrame/TheHeader.vue (hunk 2 above)
Comment (roger-in-kiva) [reply src/components/WwwFrame/TheHeader.vue]: This one was a autocomplete error

File: src/pages/Settings/SubscriptionsSettings.vue [hunk 6]
```diff
@@ -1,5 +1,8 @@
 <template>
-	<www-page class="subscriptions" :gray-background="true">
```
Comment (ryan-ludwig) [inline src/pages/Settings/SubscriptionsSettings.vue:2]: I believe we want to keep all of the `:gray-background="true"` props intact

Comment (mcstover) [reply src/pages/Settings/SubscriptionsSettings.vue:2]: Yes we do. Good call

Comment (roger-in-kiva) [reply src/pages/Settings/SubscriptionsSettings.vue:2]: Good. Gray background changes are reverted now.

File: package.json [hunk 7]
```diff
@@ -91,6 +91,7 @@
 		"instantsearch.js": "^3.6.0",
 		"isomorphic-fetch": "^3.0.0",
 		"js-cookie": "^2.2.1",
+		"jshint": "^2.13.1",
```
Comment (ryan-ludwig) [inline package.json:94]: I don't think this should be added to dependencies?

Comment (roger-in-kiva) [reply package.json:94]: Right, its a dev dependency.

File: src/pages/AutoDeposit/AutoDepositThanksPage.vue [hunk 8]
```diff
@@ -35,7 +35,7 @@
 <script>
 import gql from 'graphql-tag';
 import numeral from 'numeral';
-
+import { lightHeader } from '@/util/siteThemes';
```
Comment (ryan-ludwig) [inline src/pages/AutoDeposit/AutoDepositThanksPage.vue:38]: This can be removed

**Review by ryan-ludwig — APPROVED** [review]
Looks good! One small change then all set for me.

File: package-lock.json [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
 	"name": "kvue",
-	"version": "1.0.0",
+	"version": "2.10.0",
 	"lockfileVersion": 2,
 	"requires": true,
 	"packages": {
 		"": {
 			"name": "kvue",
-			"version": "1.0.0",
+			"version": "2.10.0",
```
Comment (mcstover) [inline package-lock.json:9]: @emuvente Guessing this is fine and will now get updated in the build/deploy process

File: src/pages/Build/BuildPage.vue [hunk 10]
```diff
@@ -1,5 +1,7 @@
 <template>
-	<www-page>
+	<www-page
+		:header-theme="greenHeader"
```
Comment (mcstover) [inline src/pages/Build/BuildPage.vue:3]: The entire "Build" section can move to a the new default light header. I think the only outlying page is the one that might have a users api credentials but that's ok for now.

Comment (mcstover) [reply src/pages/Build/BuildPage.vue:3]: We can also just let it be and wait until the full header refresh. Don't let one block the merge.

**Review by mcstover — APPROVED** [review]

File: src/pages/AutoDeposit/AutoDepositThanksPage.vue (hunk 8 above)
Comment (emuvente) [reply src/pages/AutoDeposit/AutoDepositThanksPage.vue:38]: +1

**Review by emuvente — APPROVED** [review]
Looks great!

**Pull request merged**